pymc3/math.py

```diff
@@ -186,27 +190,14 @@ def tround(*args, **kwargs):
     return at.round(*args, **kwargs)
 
 
-def logsumexp(x, axis=None, keepdims=True):
```

Comment:
Are there tests for this we can remove?

Reply:
Yeah, I guess we can remove the old tests

Reply:
Removed